README.md

```diff
@@ -23,7 +22,7 @@
 
 <p align="center">
   <a href="https://opencollective.com/docsify/order/3254">
-    <img src="https://opencollective.com/docsify/tiers/gold-sponsor.svg?avatarHeight=36">
```

Comment:
I think we could keep this as a quick button for other sponsors. who wanna be a `Sponsor`.

Reply:
This is just a showcase of Gold Sponsors.

Reply:
> This is just a showcase of Gold Sponsors.

iiuc, it is a `To be a Sponsor` button.